service/btrfs-balance.service

```diff
@@ -3,7 +3,7 @@ Description=btrfs balance service
 
 [Service]
 Type=simple
-ExecStart=/usr/bin/keepalive-tool /usr/sbin/btrfs-balance
+ExecStart=/bin/dbus-send --system --print-reply --dest=org.nemomobile.BtrfsBalancer /BtrfsBalancer org.nemomobile.BtrfsBalancer.maintenance int32:75 int32:25
```

Comment:
Won't there be a timeout when waiting for a reply if the timeout operation takes long? Or if it returns immediately, won't that look to systemd like the service is finished when in fact it's still running?